packages/core/utils/package.json

```diff
@@ -3,35 +3,40 @@
   "version": "1.11.9",
   "description": "Medusa utilities functions shared by Medusa core and Modules",
   "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/medusajs/medusa",
     "directory": "packages/utils"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=20.0.0"
   },
   "publishConfig": {
     "access": "public"
   },
   "files": [
-    "dist"
+    "dist",
+    "!dist/**/__tests__",
+    "!dist/**/__mocks__",
+    "!dist/**/__fixtures__"
```

Comment:
Now that I am type checking the tests as well using the TypeScript compiler they will be bundled in the output.

However, we can ignore them from the publish list of `files` that makes their way to npm.

Another alternative is to have a different `tsconfig.json` file for the tests. However, the only downside of doing that is the code editors not reading that file.